api/users/serializers.py

```diff
@@ -77,7 +77,32 @@ class UserSerializer(JSONAPISerializer):
     writeable_method_fields = frozenset([
         'accepted_terms_of_service',
     ])
-    non_anonymized_fields = ['type']
+    anonymized_fields = [
```

Comment:
This one concerns me a bit. Most of the models it doesn't matter so much, but if this one doesn't do what it's supposed to, then it will almost certainly reveal information. What would you think about modifying the test for this to verify that we only get a type rather than just that the `id`s are removed?

Reply:
that's a great idea, to make sure that this serializer is protected!